### DISCUSSION TITLE
唉？突然有个想法

### DISCUSSION THREAD

**Discussion by JobNumber-9527** [share121/inter-knot#2560] [其他]
既然这里是发委托的地方。那有没有b站的代肝来打点广告呢<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by Yoimiya-M** [comment]
你别说，你还真别说<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by 00zzxx** [comment]
有吧，看见过一两个，不过都很快被举报掉了_(:зゝ∠)_
不过大家水贴也挺快的，被刷下去导致一时半会上不来也有可能
